 # 自律移動先座標
 float32 goal_position_x
 float32 goal_position_y
 float32 goal_position_degree
